### ISSUE TITLE
Proftpd - Roadmap

### ISSUE THREAD

**Issue by Shailendranjain** [proftpd/proftpd#823]
### What I Did

When I tried compiling the source for proftpd (version 1.3.3g) with mysql 8, I received an error related data type 'my_bool'. (which need to be changed to bool).
Eeven in the latest proftpd version, the same issue seem to be existing.

### What I Expected/Wanted

I thought the latest proftpd version would support mysql8. 

Questions 👍 
 
  1. What is the road map for proftpd ?

  2. Will there be any new releases in near future ( say within next six months) ?

  3. If  mysql8 is not compatible, I was thinking to switch to PostgreSQL. What is the latest PostgreSQL version that proftpd supports ?

### ProFTPD Version and Configuration

I use the basic configuration file (proftpd.conf) and it is reproducible.

Thanks,
Shailendra

**Comment by Ashot000** [comment]
Hi,
Any Updates here ?

**Comment by Castaglia** [comment]
Can you see if the issue still persists with the latest releases, either 1.3.6a or 1.3.7rc1?

**Comment by snjain** [comment]
Can you please let me know when will the official release be made?

**Comment by snjain** [comment]
Is the latest release 1.3.6a considered to be official stable release?

**Comment by Ashot000** [comment]
I am using now 1.3.7rc1 and seem all working well.

**Comment by Castaglia** [comment]
@snjain yes, 1.3.6a is considered the current stable release.  ProFTPD release versioning is described in more detail [here](http://www.proftpd.org/docs/howto/Versioning.html).

**Comment by snjain** [comment]
Did a quick check on 1.3.6a code base and looks like mysql8 issue is still existing.

Can you please advise what is the likely date for 1.3.7 official release

**Comment by snjain** [comment]
Also is there any possibility that MySQL8 support be provided for 1.3.6a until 1.3.7 becomes stable release?

**Comment by Castaglia** [comment]
You can file a separate issue, requesting support for MySQL 8.  That's the best way to go about it.

And no, I cannot provide dates, estimated or otherwise, for official releases.
